plugins/chain_api_plugin/chain_api_plugin.cpp

```diff
@@ -41,6 +41,45 @@ parse_params<chain_apis::read_only::get_transaction_status_params, http_params_t
    }
 }
 
+// if actions.data & actions.hex_data provided, use the hex_data since only currently support unexploded data
```

Comment:
- As part of this effort, I think we should support exploded `data`.
- As part of this effort, I think we should provide optional support for exploded `data` in cleos. For cleos, this should be optional as a user may want to prevent cleos from communicating with nodeos. Could support the option of providing abi. See `--unpack-action-data` option `--abi-file`.
- To increase security, if both "data" and "hex_data" are provide then we should verify they are equivalent.
- Please add some tests.

Reply:
I added a separate issue to include exploded data handling improvements:
https://github.com/AntelopeIO/leap/issues/1282